src/one_dragon/utils/cv2_utils.py

```
@@ -1234,3 +1235,54 @@ def is_colorful(img: MatLike, saturation_threshold: int = 30, color_ratio_thresh
+        在游戏下方的对话框中, 应该是只有文字边缘像素颜色值会不在50以下
+    2.  统计RGB之差小于 rgb_diff_threshold 的像素占比, 以判断是否为灰度像素
+
+    Args:
+        img: 输入图像（RGB格式）
+        rect: 先裁切
+        threshold: 颜色阈值，RGB都低于此或者都高于(255-此)认为接近黑白
+        rgb_diff_threshold: 颜色阈值，RGB只差小于此则认为像素块为灰色
+        percent_rate: 满足条件的点的占比阈值
+
+    Returns:
+        bool: True 表示是彩色的，False 表示接近灰度
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)